pkg/controller/batchrelease/workloads/deployment_canary_control_plane.go

```diff
@@ -62,46 +62,10 @@ func NewDeploymentRolloutController(cli client.Client, recorder record.EventReco
 
 // VerifyWorkload verifies that the workload is ready to execute release plan
 func (c *DeploymentsRolloutController) VerifyWorkload() (bool, error) {
```

Comment:
is the VerifyWorkload needed after the changes ? If yes, the name "VerifyWorkload" seems improper now.